### PR TITLE
Rename MessageInterface.getName to MessageInterface.getHandlerName 

### PR DESCRIPTION
**Pull request by stanislav4er**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #112

### CODE REVIEW

File: src/Queue.php [hunk 1]
```diff
@@ -43,14 +43,17 @@ public function push(MessageInterface $message): void
...
+        );
 
         /** @psalm-suppress PossiblyNullReference */
         $this->adapter->push($message);
 
         $this->logger->debug(
-            'Successfully pushed message "{name}" to the queue.',
-            ['name' => $message->getName()]
+            'Successfully pushed message with handler name "{handlerName}" to the queue. Assigned ID #{id}.',
+            ['name' => $message->getHandlerName(), 'id' => $message->getId()]
```
Comment (viktorprogger) [inline src/Queue.php:56]: Hi! Id can be `null` here, so for this situation to be clear in the log message I suggest to do like this:
```suggestion
            ['name' => $message->getHandlerName(), 'id' => $message->getId() ?? 'null']
```

**Review by viktorprogger — CHANGES_REQUESTED** [review]

**Comment by viktorprogger** [comment]
Everything other LGTM :+1:

**Comment by stanislav4er** [comment]
MB. Fixed.

File: src/Queue.php (hunk 1 above)
Comment (stanislav4er) [reply src/Queue.php]: MB. Fixed.

Comment (stanislav4er) [reply src/Queue.php]: Should I press "Resolve conversation"?

**Comment by viktorprogger** [comment]
Thanks :+1:

**Pull request merged**